README.md

```diff
@@ -138,3 +138,52 @@ HTTP Response:
     "count": int
 }
 ```
+
+## Adding Help Request
+
+POST "/help_request"
+
+Request Body:
+
+```
+{
+    "user_id": string
```

Comment:
Frontend has never used snake_case up to this point. A lot of the other endpoints take request data in camelCase too

Reply:
Changed to camel. Check other attributes as well